src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/Offence.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models
+
+import java.time.LocalDate
+
+data class Offence(
+  val date: LocalDate?,
+  val code: String?,
+  val description: String?,
+)
```

Comment:
I think these can be non-nullable based on the fact that we're only pulling from NOMIS right now, and these fields in NOMIS are not nullable

Reply:
Agreed